robot/Cumulus/tests/browser/gift_entry/reorder_template.robot

```diff
@@ -0,0 +1,79 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/NPSPSettingsPageObject.py
+...             robot/Cumulus/resources/GiftEntryPageObject.py
+Suite Setup     Run keywords
+...             Open Test Browser
+...             API Check And Enable Gift Entry
+Suite Teardown  Capture Screenshot and Delete Records and Close Browser
+
+*** Keywords ***
+
+Get Template Builder Field Names
+  @{builder_form_fields} =  Get Webelements  xpath://lightning-input[@class="slds-truncate slds-form-element"]//label
+  ${builder_labels} =  Create List
+
+  FOR  ${label}  IN  @{builder_form_fields}
+      ${name} =  Get Text  ${label}
+      Append to List  ${builder_labels}  ${name}
+  END
+  Set Suite Variable  ${builder_labels}
+
+
+Get Template Form Field Names
+  @{gift_form_fields} =  Get Webelements  xpath://label[@class="slds-form-element__label" or @class="slds-form-element__label slds-no-flex"]
```

Comment:
Same as my above comment. Move to to specific locators file

Reply:
Moved this to a keyword as well.